### ISSUE TITLE
function_demo/mespnow doesn't exist yet is referenced in the documentation

### ISSUE THREAD

**Issue by will-emmerson** [espressif/esp-mdf#208]
https://docs.espressif.com/projects/esp-mdf/en/latest/api-reference/mespnow/index.html#application-examples links to non-existent folder. There are examples for all the other esp-mdf modules such as mwifi, mconfig but not mespnow.

**Comment by EspHuifeng** [comment]
Yes, there is no example that shows how to use mespnow.  If you want to know how to use mespnow. you can reference components/mdebug/mdebug_espnow.c

**Comment by will-emmerson** [comment]
Thanks, I managed to get it working. That reference isn't a very good example because it's not that obvious what the parameters are doing, a seperate example would be better.
